src/main/steps/common/yourhearings/hearings/utils.ts

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { Case } from 'app/case/case';
+import dayjs from 'dayjs';
+
+import { Attendee, CaseType, HearingsList, PartyType, hearingDay } from '../../../../app/case/definition';
+import { AppRequest } from '../../../../app/controller/AppRequest';
+import { HEARING_METHOD } from '../../../../steps/constants';
+import { getCasePartyType } from '../../../../steps/prl-cases/dashboard/utils';
+
+import { cy, en } from './content';
+
+export function getHearingMethod(req: AppRequest, attendees: Attendee[]): string {
```

Comment:
change it to getHearingDetails

Reply:
this method is retuening method type like in person, audio, video, hence the name